src/wagtail_ai/migrations/0002_populate_default_prompts.py

```diff
@@ -0,0 +1,48 @@
+# Generated using Django 4.2.7 on 2023-12-01 09:35
+
+from django.conf import settings
+from django.db import migrations
+
+import wagtail_ai
+
+
+def set_default_ai_prompts(apps, schema_editor):
+    """
+    Loop through each default prompt in settings and
+    create instances in the database.
+    """
+    Prompt = apps.get_model("wagtail_ai", "Prompt")
+
+    default_prompts = (
+        wagtail_ai.DEFAULT_PROMPTS if hasattr(wagtail_ai, "DEFAULT_PROMPTS") else []
+    )
+
+    for default_prompt in (
+        getattr(settings, "WAGTAIL_AI_PROMPTS", []) or default_prompts
+    ):
+        Prompt.objects.create(
+            label=default_prompt["label"],
+            prompt=default_prompt["prompt"],
+            description=default_prompt.get("description", ""),
+            method=default_prompt.get("method", None),
+        )
+
+
+def reverse_set_default_ai_prompts(apps, schema_editor):
+    """
+    Reverse the migration by deleting all instances of the Prompt model.
+    """
+    Prompt = apps.get_model("wagtail_ai", "Prompt")
+    Prompt.objects.all().delete()
```

Comment:
**Issue:** You are deleting all the prompts added by the users, not just the one added by the migration so this reverse migration is incorrect. I suggest we treat this as a `migrations.RunPython.noop` instead. Better safe than sorry.